workspaces/api-gateway/src/api/v1/v1.module.ts

```diff
@@ -1,18 +1,34 @@
+import Stripe from 'stripe';
 import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
 import type { RouteTree } from '@nestjs/core';
 
 import { ProductsModule } from 'api/v1/products/products.module';
 import { CategoriesModule } from 'api/v1/categories/categories.module';
 import { V1Profile } from 'api/v1/v1.profile';
+import { WebhooksController } from 'api/v1/webhook.controller';
+import { PaymentsModule } from 'api/v1/payments/payments.module';
+import { Payment } from 'database/models/payment.entity';
+import { StripeMethod } from 'database/models/paymentMethods/stripeMethod.entity';
+import { PaymentDetail } from 'database/models/paymentDetail.entity';
+import { CodMethod } from 'database/models/paymentMethods/codMethod.entity';
+import { CartsModule } from 'api/v1/carts/carts.module';
 
 @Module({
-  imports: [ProductsModule, CategoriesModule],
-  providers: [V1Profile],
+  imports: [
+    TypeOrmModule.forFeature([Payment, PaymentDetail]),
```

Comment:
why we need to import this at `V1` module?

Reply:
chỗ này em import dư cái Payment Detail, còn Payment em dùng ở webhook controller nên phải import ở V1 module ạ

Reply:
then why don't we import it at `webhook module`?